kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -315,6 +325,37 @@ public String createConsumer(String group, ConsumerInstanceConfig instanceConfig
     executor.submit(new RunnableReadTask(new ReadTaskState(task, state, callback)));
   }
 
+  class ReadFutureTask<V> extends FutureTask<V> {
+
+    private Runnable myTask;
+
+    public ReadFutureTask(Runnable runnable, V result) {
+      super(runnable, result);
+      this.myTask = runnable;
+    }
+
+    public Runnable getReadTask() {
```

Comment:
Since this class is meant to wrap `RunnableReadTask`, I think this method should return `RunnableReadTask` directly

Reply:
ok,i will accept you suggest

Reply:
> ass is meant to wrap `RunnableReadTask`, I think
hi,do you think to modify like this ?: 
``` java
class MyFutureTask<V> extends FutureTask<V> {

    private Runnable myTask;

    private Callable callable;

    private V result;


    public MyFutureTask(Runnable runnable, V result) {
        super(runnable, result);
        this.myTask = runnable;
        this.result = result;
    }

    public MyFutureTask(Callable callable) {
        super(callable);
     }

    public Read getMyTask() {
        return (Read) myTask;
    }

    public V getResult() {
        return result;
    }
}
``` 
in the MyFutureTask,the class is just for this executor,so wo cast the runnable to the class directly

Reply:
I don't understand, `MyFutureTask` is not in the committed changes.
My idea is to have only one cast in lines 132-134.
Current:
```
                ReadFutureTask readFutureTask = (ReadFutureTask) r;
                RunnableReadTask readTask = (RunnableReadTask)readFutureTask.getReadTask();
```
Proposed:
```
                RunnableReadTask readTask = ((ReadFutureTask) r).getReadTask();
```

Reply:
ok,i unstand it .